packages/node/test/integration/install-virtual.spec.ts

```diff
@@ -23,7 +23,7 @@ import {
 } from "./utils";
 
 describe("Node method follows spec - proposeInstallVirtual", () => {
-  jest.setTimeout(30000);
+  jest.setTimeout(60000);
```

Comment:
lower?

Reply:
I can experiment with lower timeouts, just doubled it because the test is twice as long now.